app/controllers/provider_foreman_controller.rb

```diff
@@ -139,12 +139,13 @@ def update_authentication_provider(mode = :validate)
   end
 
   def build_credentials
-    creds = {}
-    if params[:log_userid]
-      default_password = params[:log_password] ? params[:log_password] : @provider_cfgmgmt.authentication_password
-      creds[:default] = {:userid => params[:log_userid], :password => default_password}
-    end
-    creds
+    return {} if params[:log_userid].nil?
```

Comment:
technically, this is more consistent with prior behavior for false values:  `return {} unless params[:log_userid]`